rbc/src/test/java/com/library/rbc/controller/bookController/BookControllerSetUp.java

```diff
@@ -0,0 +1,50 @@
+package com.library.rbc.controller.bookController;
```

Comment:
Package names should not be `camelCase`. See [naming packages](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html).

Usually we would go with all lowercase, no hyphens, `bookcontroller` in this case. Please check and correct all package names.